js/app.js

```diff
@@ -85,16 +85,35 @@ function canFactionBePicked(faction, selectedFactions) {
 
 function randomizeFactions() {
   const availableFactions = Array.from(DATA.FACTION_LIST_BY_REACH);
-  const numPlayers = State.playerList.length;
+  const humanPlayers = State.playerList.length;
+  const useBot = document.getElementById("use-bot")
+
+  if(humanPlayers == 1) {
```

Comment:
This will force the element to be checked right? Not sure I like this, I think the checking of the box should be done by the user.